superset/models/dashboard.py

```diff
@@ -254,7 +254,6 @@ def data(self) -> Dict[str, Any]:
     @cache.memoize(
         # manage cache version manually
         make_name=lambda fname: f"{fname}-v2.1",
-        timeout=config["DASHBOARD_CACHE_TIMEOUT"],
```

Comment:
This cache will use the global default timeout from `CACHE_CONFIG` as I don't see a strong reason to give a different timeout.